standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParserSuiteTemplate.scala

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization.interpreter.stages
+
+import java.security.InvalidParameterException
+import java.sql.{Date, Timestamp}
+
+import org.apache.log4j.{LogManager, Logger}
+import org.apache.spark.sql.types._
+import org.scalatest.FunSuite
+import za.co.absa.enceladus.standardization.interpreter.dataTypes.ParseOutput
+import za.co.absa.enceladus.utils.types.Defaults
+import za.co.absa.enceladus.standardization.interpreter.stages.TypeParserSuiteTemplate._
+
+trait TypeParserSuiteTemplate extends FunSuite {
+  protected def createCastTemplate(toType: DataType, pattern: String, timezone: Option[String]): String
+
+  private val sourceFieldName = "sourceField"
+  private val dateEpochPattern = "epoch"
+  private val timestampEpochPattern = "epochmilli"
```

Comment:
Why redefine epoch constants here if they are already defined as `EpochKeyword` and `EpochMilliKeyword`?